ssl/ssltest.c

```diff
@@ -1159,6 +1254,46 @@ int main(int argc, char *argv[])
             if (--argc < 1)
                 goto bad;
             alpn_expected = *(++argv);
+        } else if (strcmp(*argv, "-server_sess_out") == 0) {
+            if (--argc < 1)
+                goto bad;
+            server_sess_out = *(++argv);
+        } else if (strcmp(*argv, "-server_sess_in") == 0) {
+            if (--argc < 1)
+                goto bad;
+            server_sess_in = *(++argv);
+        } else if (strcmp(*argv, "-client_sess_out") == 0) {
+            if (--argc < 1)
+                goto bad;
+            client_sess_out = *(++argv);
+        } else if (strcmp(*argv, "-client_sess_in") == 0) {
+            if (--argc < 1)
+                goto bad;
+            client_sess_in = *(++argv);
+        } else if (strcmp(*argv, "-should_reuse") == 0) {
+            if (--argc < 1)
+                goto bad;
+            should_reuse = !!atoi(*(++argv));
+        } else if (strcmp(*argv, "-no_ticket") == 0) {
+            no_ticket = 1;
+        } else if (strcmp(*argv, "-no-srv-ssl2") == 0) {
+            srv_options |= SSL_OP_NO_SSLv2;
+        } else if (strcmp(*argv, "-no-srv-ssl3") == 0) {
+            srv_options |= SSL_OP_NO_SSLv3;
+        } else if (strcmp(*argv, "-no-srv-tls1") == 0) {
+            srv_options |= SSL_OP_NO_TLSv1;
+        } else if (strcmp(*argv, "-no-srv-tls1_1") == 0) {
+            srv_options |= SSL_OP_NO_TLSv1_1;
+        } else if (strcmp(*argv, "-no-srv-tls1_2") == 0) {
+            srv_options |= SSL_OP_NO_TLSv1_2;
+        } else if (strcmp(*argv, "-no-srv-dtls1") == 0) {
+            srv_options |= SSL_OP_NO_DTLSv1;
+        } else if (strcmp(*argv, "-no-srv-dtls1_2") == 0) {
+            srv_options |= SSL_OP_NO_DTLSv1_2;
+        } else if (strcmp(*argv, "-should_negotiate") == 0) {
+            if (--argc < 1)
+                goto bad;
+            should_negotiate = *(++argv);
```

Comment:
You mean only that one part, right?  Not the rest of this which seems like a bugfix (correct me if I'm wrong).  And then, does adding this could make it possible to test the bugfix?